lib/registries/addon/services/brand.ts

```diff
@@ -0,0 +1,25 @@
+import Service from '@ember/service';
+import Brand from 'ember-osf-web/models/brand';
+
+export default class BrandService extends Service {
+    currentBrand: Brand? = null;
+
+    setBrand(brand: Brand) {
+        this.set('currentBrand', brand);
+        this.setCssVariables();
+    }
+
+    setCssVariables() {
+        if (!this.currentBrand) {
+            return;
+        }
+
+        const documentStyle = document.documentElement.style;
+
+        documentStyle.setProperty('--primary-color', this.currentBrand.primaryColor);
+        documentStyle.setProperty('--secondary-color', this.currentBrand.secondaryColor);
+        documentStyle.setProperty('--navbar-logo-img-url', this.currentBrand.navbarLogoImage);
+        documentStyle.setProperty('--hero-logo-img-url', this.currentBrand.heroLogoImage);
+        documentStyle.setProperty('--hero-background-img-url', this.currentBrand.heroBackgroundImage);
```

Comment:
how would you feel about setting the css variables in an element modifier? in the application template we could do something like:
```hbs
<div
  data-test-application
  ...
  {{with-branding brand=this.brandService.currentBrand}}
>
```
wouldn't need to remember to use `setBrand`, and might give us nice flexibility on how we choose to apply brands now and in the future

Reply:
I like it! Will start implementing in that direction